cms/templates/js/course-outline.underscore

```diff
@@ -30,10 +30,26 @@ if (statusType === 'warning') {
     statusIconClass = 'fa-lock';
 }
 
-var gradingType = gettext('Not Graded');
+var gradingType = gettext('Ungraded');
```

Comment:
I believe this came from @marcotuts . In any case, I'll ping @mhoeber for doc references.